lib/app/components/pages/create_chat_group/views/create_chat_group_view.dart

```diff
@@ -22,47 +22,107 @@ class CreateChatGroupView extends GetView<CreateChatGroupController> {
         children: [
           Padding(
             padding: const EdgeInsets.all(8.0),
-            child: SearchBar(
-              onChanged: (text) {
-                if (text != null) controller.searchUser(text);
+            child: Focus(
+              onFocusChange: (hasFocus) {
+                controller.isFocusedOnSearchBar.value = hasFocus;
               },
-            ),
-          ),
-          Expanded(
-            child: Obx(
-              () => ListView.builder(
-                itemCount: controller.searchResult.length + 1,
-                itemBuilder: (context, index) {
-                  if (index == 0) {
-                    return controller.searchResult.isNotEmpty
-                        ? const Divider()
-                        : Container();
-                  } else {
-                    return Column(
-                      children: [
-                        SeachResultListTile(
-                          index: index,
-                          text: controller.searchResult.elementAt(index - 1),
-                          onPressed: () =>
-                              controller.searchResult.removeAt(index - 1),
-                        ),
-                        const Divider(),
-                      ],
-                    );
-                  }
+              child: SearchBar(
+                onChanged: (text) {
+                  if (text != null) controller.searchUser(text);
                 },
               ),
             ),
           ),
+          //検索バーフォーカス時はSeachResultListTile、フォーカスが外れている場合はFriendCardを表示
+          Obx(
+            () => controller.isFocusedOnSearchBar.value
+                ? Expanded(
+                    child: ListView.builder(
+                      itemCount: controller.searchResult.length + 1,
```

Comment:
```suggestion
                      itemCount: controller.searchResult.length,
```
この辺の辻褄あわせみたいなのがちょっとよくわかってないですが、
+1や-1しなくても良さげなので特に問題なければなしにしてほしいです。

Reply:
@king-kazu39 
少し分かりにくくて恐縮ですが、下記に説明するように必要な処理なので残したいです。

itemCountの+1分は最も上にあるSearchResultListTileの上下のDividerのうち上側のDividerを表示するためのものです。

itemBuilderでループさせているのはSearchResultListTile＋タイルの下側のDividerであるため、ループの初回(ループにおけるindex=0)でDividerを表示してやらないと一番上に表示されているSearchResultListTileだけ区切り線がない表示となってしまいます。

ただし、検索結果がない場合はDividerを一切表示したくないため、ループの初回で検索結果が0件かどうかで分岐させています。0件なら空のContainerを、それ以外ならDividerを返す処理となっています。

Reply:
@king-kazu39 
「index-1」のような無くした書き方ができたので、修正しpushし直しました。itemBuilderでreturnするColumnのchildrenプロパティのListの中でもif文を利用できることがわかったので、childrenの中身を下記のような構成に修正することで、「index-1」を使わずとも処理の記述ができました。

Divider()　←index=0、かつ検索結果が1件以上の場合のみColumnのchildrenに含める
SearchResultListTile　←常にColumnのchildrenに含める
Divider()　←常にColumnのchildrenに含める